src/lib/Datastore.js

```diff
@@ -37,6 +37,26 @@ export default class Datastore {
     }
 
     /**
+    * Set the replay lock key (if it does not already exist) with an expiration
+    * of fifteen minutes
+    * @return {undefined}
+    */
+    setLock() {
+        const ttl = 60 * 15; // fifteen minutes
```

Comment:
Datastore method should be generic. You can call this setExpire which takes a key, value and ttl as params.